features/steps/cli_steps.py

```diff
@@ -150,6 +150,14 @@ def create_configuration_file(context):
         yaml.dump(config, config_file, default_flow_style=False)
 
 
+@given("I have fixed logs write permission")
+def modify_write_permission(context):
```

Comment:
A bit of explanation would help. Something like:

```suggestion
def modify_write_permission(context):
def modify_write_permission(context):
    """
    Kedro-docker mounts some subdirectories the current directory (like logs, notebooks etc) into the Docker container. If you run kedro commands with different users, they might create files and directories not writable by each other. Fixing the permissions here.
    """
```